content/teknologi/altinnstudio/altinn-api/_index.md

```diff
@@ -7,1012 +7,61 @@ aliases:
+The Platform API are primarily made to support the applications hosted on the platform, but all GET methods can be used directly by both application owners and users.
 
-```http
-GET {storagePath}/instances/347829/762011d1-d341-4c0a-8641-d8a104e83d30/data/692ee7df-82a9-4bba-b2f2-c8c4dac69aff
-```
-
-```json
-{
-    "data": [
-        {
-            "id": "692ee7df-82a9-4bba-b2f2-c8c4dac69aff",
-            "dataType": "default",
-            "filename": "default.xml",
-            "contentType": "application/xml",
-            "lastChange": "2019-03-06T15:00:23Z",
-            "lastChangedBy": "org24",
-            "size": 34059,
-            "locked": false,
-            "applicationOwner": {
-                "downloaded": ["2019-05-15T08:23:01Z"]
-            }
-        }
-    ]
-}
-```
-
-## Receipt data element
-
-All applications has an element type called `ref-data-as-pdf`. This is the place to store generated pdf receipt data.
-
-### Upload a pdf and associate the pdf with an existing data element
-
-```http
-POST {storagePath}/instances/347829/762011d1-d341-4c0a-8641-d8a104e83d30/data?dataType=ref-data-as-pdf&refs=692ee7df-82a9-4bba-b2f2-c8c4dac69aff
-```
-
-Results in a new data element with reference to the first. It is also possible to reference multiple data elements.
-
-```json {linenos=false,hl_lines=[4,25]}
-{
-    "data": [
-        {
-            "id": "692ee7df-82a9-4bba-b2f2-c8c4dac69aff",
-            "dataType": "model1",
-            "filename": "default.xml",
-            "contentType": "application/xml",
-            "lastChanged": "2019-03-06T15:00:23Z",
-            "lastChangedBy": "org24",
-            "size": 34059,
-            "locked": true,
-            "applicationOwner": {
-                "downloaded": ["2019-05-15T08:23:01Z"]
-            }
-        },
-        {
-            "id": "c15f0401-e19d-4f1d-8ad1-1ce8cc96eb5d",
-            "dataType": "ref-data-as-pdf",
-            "filename": "model1-view.pdf",
-            "contentType": "application/pdf",
-            "lastChanged": "2019-10-10T15:00:23Z",
-            "lastChangedBy": "org24",
-            "size": 34059,
-            "locked": false,
-            "refs": ["692ee7df-82a9-4bba-b2f2-c8c4dac69aff"]
-        }
-    ]
-}
-```
-
-### Download the PDF of a data element
-
-Select the data PDF you want to download from the list of the instances' list of data elements.
-
-Get list
-
-```http
-GET {storagePath}/instances/347829/762011d1-d341-4c0a-8641-d8a104e83d30/data
-```
+The storage API provides access to all instances across all applications. It can be used to access metadata about applications, instances, data elements and events, as well as the actual data content. This API should be the preferred method for application owners to download data associated with instances created based on their applications, and for user applications with a form of message box and the need to retrieve archived instances and their data.
```

Comment:
Should we refer to storage with capital S to indicate that is it Platform Storage ? 

Regarding events in the list of the metadata to be accessed. how can we make it clear that it it's Platform Events events, but instanceEvents we are refering to?

Reply:
The last sentence is a bit word heavy.. Is there a way it can be restructured so that it is easier to read?